Qadir__Work/JavaScript/Assignment12/Question2/employeeClassData.js

```diff
@@ -0,0 +1,71 @@
+class Employee {
+    constructor(empno, name, address, city, salary){
+        this.empno = empno;
+        this.name = name;
+        this.address = address;
+        this.city = city;
+        this.salary = salary;
+        this.da;
+        this.hra;
+        this.pf;
+        this.gross;
+        this.net;
+        this.basic;
+        this.counter = 0;
```

Comment:
define an empty constructor and use getData method to read values from the user

Reply:
ok Mam